accessibility-checker-engine/help-v4/en-US/area_alt_exists.html

```diff
@@ -60,11 +60,11 @@ <h3 id="ruleMessage"></h3>
 <img src="welcome.gif" usemap="#map1" alt="Areas in the library. Select an area for more information on that area." />
 <map id="map1" name="map1">
 <area shape="rect" coords="0,0,30,30" href="reference.html" alt="Reference" />
-<area shape="rect" coords="34,34,100,100" href="media.html" alt="Audio visual lab" />
+<area shape="rect" coords="34,34,100,100" href="media.html" alt="Audiovisual lab" />
 </map>
 ```
 
-Note: Example code includes material copied from or derived from [WCAG technique H24: Providing text alternatives for the area elements of image maps](https://www.w3.org/WAI/WCAG22/Techniques/html/H24) Copyright © [2013-2017] W3C® (MIT, ERCIM, Keio, Beihang).
+Note: Example code includes material copied from or derived from [H24: Text alternatives for the area elements of image maps](https://www.w3.org/WAI/WCAG22/Techniques/html/H24) Copyright © [2013-2017] W3C® (MIT, ERCIM, Keio, Beijing).
```

Comment:
Beihang was correct.
```suggestion
Note: Example code includes material copied from or derived from [H24: Text alternatives for the area elements of image maps](https://www.w3.org/WAI/WCAG22/Techniques/html/H24) Copyright © [2013-2017] W3C® (MIT, ERCIM, Keio, Beihang).
```